test/channels_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:rtchat/models/channels.dart';
+
+// This is the very first time I write a test in flutter. take it easy on me plz! :D
+void main() {
+  test('Create Channels Test', () {
```

Comment:
The string here should describe the test, so in this case something like

```
test('== compares provider and channelId', () {
   ...
});
```

and same for the test below.